en/flight_controller/cuav_v5_plus.md

```diff
@@ -70,7 +70,7 @@ Download **V5+** pinouts from [here](http://manual.cuav.net/V5-Plus.pdf).
 
 ## Voltage Ratings
 
-*V5+ AutoPilot* support redundant power supply - up to three sources may be used: `Power1`, `Power2` and `USB`.
+*V5+ AutoPilot* supports redundant power supply - up to three sources may be used: `Power1`, `Power2` and `USB`.
 You must supply power to at least one of these sources, or the flight controller will be unpowered.
 
 > **Note** The *output* power sources **FMU PWM OUT** and **I/O PWM OUT** (0V to 36V) cannot be used to power the flight controller board, and are not powered by it. 
```

Comment:
Again this is speaking about the servo rail it is not an output it is neither an inport providing a voltage to the FMU or an output it is only  monitored rail and can only bec

- [ ] used with Abe